ogre/src/OgreMaterial.cc

```diff
@@ -44,21 +44,53 @@ void OgreMaterial::Destroy()
 {
   if (!this->Scene()->IsInitialized())
     return;
-
+  std::string materialName;
   Ogre::MaterialManager &matManager = Ogre::MaterialManager::getSingleton();
 #if OGRE_VERSION_LT_1_10_1
   if (!this->ogreMaterial.isNull())
   {
+    materialName = this->ogreMaterial->getName();
+
+    this->ogreTexState->setBlank();
+    auto indexUnitStateToRemove =
+      this->ogrePass->getTextureUnitStateIndex(this->ogreTexState);
+    this->ogrePass->removeTextureUnitState(indexUnitStateToRemove);
+
     matManager.remove(this->ogreMaterial->getName());
     this->ogreMaterial.setNull();
-  }
```

Comment:
I think this is removed by accident. 

Windows build caught a compile error:
C:\Jenkins\workspace\ign_rendering-pr-win\ws\ign-rendering\ogre\src\OgreMaterial.cc(94,1): error C2059: syntax error: '}'